dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/startup/InitTableAlteror.java

```diff
@@ -150,6 +150,10 @@ public void execute()
         executeSql( "UPDATE programstageinstance SET lastsynchronized = to_timestamp(0) WHERE lastsynchronized IS NULL" );  //Do not remove this line if some cleanup will ever happen
         executeSql( "alter table programstageinstance alter column lastsynchronized set not null" );
         executeSql( "alter table programstageinstance alter column lastsynchronized set default to_timestamp(0)" );
+
+        // Update trackedentityattribute set skipsynchronization = false where skipsynchronization = null
+        executeSql( "UPDATE programstagedataelement SET skipsynchronization = false WHERE skipsynchronization IS NULL" );
+        executeSql( "alter table programstagedataelement alter column skipsynchronization set not null" );
```

Comment:
Although some people use lowercase and some uppercase for SQL terms, try to use the same; If you ask me, uppercase is better :)